Exercises-4-OOP/src/ExercisesOnClasses/RectangleTest.java

```diff
@@ -0,0 +1,27 @@
+package ExercisesOnClasses;
+
+public class RectangleTest {
+
+	public static void main(String[] args) {
+		Rectangle r1 = new Rectangle(1.2f, 3.4f);
+		String message = r1.toString();
+		System.out.println(r1); // toString()
```

Comment:
The test is showing you what is tested, so for example, here it says that `toString()` method is tested.
Use these hints.